### ISSUE TITLE
[T3] [Accessibility] [Functional] Unable to navigate using Tab 

### ISSUE THREAD

**Issue by naveennayakp** [Mohanme13/Feb-19-Bug-bash#155]
In family tab, unable to navigate from Self column using Tab
